main/integration-tests/support/resources/users/user.js

```diff
@@ -38,6 +38,10 @@ class User extends Resource {
   }
 
   // ************************ Helpers methods ************************
+
+  async updateUserPassword(password) {
```

Comment:
I would recommend renaming it to 'updatePassword()'  since adding 'user' to the method name is redundant (since it is the User resource).